src/sentry/static/sentry/app/components/events/interfaces/breadcrumbsV2/breadcrumbData/breadcrumbDataHttp.tsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import omit from 'lodash/omit';
+
+import ExternalLink from 'app/components/links/externalLink';
+import {getMeta} from 'app/components/events/meta/metaProxy';
+import {t} from 'app/locale';
+import {defined} from 'app/utils';
+
+import getBreadcrumbCustomRendererValue from '../../breadcrumbs/getBreadcrumbCustomRendererValue';
+import {BreadcrumbTypeHTTP} from '../../breadcrumbs/types';
+import BreadcrumbDataSummary from './breadcrumbDataSummary';
+
+type Props = {
+  breadcrumb: BreadcrumbTypeHTTP;
+};
+
+const BreadcrumbDataHttp = ({breadcrumb}: Props) => {
+  const {data} = breadcrumb;
+
+  const renderUrl = (url: any) => {
+    if (typeof url === 'string') {
+      return url.match(/^https?:\/\//) ? (
+        <ExternalLink data-test-id="http-renderer-external-link" href={url}>
+          {url}
+        </ExternalLink>
+      ) : (
+        <span>{url}</span>
+      );
+    }
+
+    try {
+      return JSON.stringify(url);
+    } catch {
+      return t('Invalid URL');
```

Comment:
Should we emit the original value here instead of an error message?

Reply:
I am not sure (I have to check), but I have the feeling that if we return the original value here, we might cause some error